### ISSUE TITLE
flatbuffer_direct INT8 pipeline miscalibrates input/output tensors (possibly related to #724)

### ISSUE THREAD

**Issue by IsliBasha** [PINTO0309/onnx2tf#942]
### onnx2tf version number

2.5.0

### Description

Converting `dima806/house-plant-image-detection` (a ViT model from Hugging Face) with `output_integer_quantized_tflite=True` + `custom_input_op_name_np_data_path` (40 calibration images) produces a `uint8`-in/`uint8`-out `.tflite` file with badly miscalibrated quantization parameters, and a file size that barely differs from the un-quantized float32 model.

**Note before the details:** this may be the same root cause as #724 (bloated full-integer-quant files traced to too-few calibration images). I tried reproducing with more calibration images (64 and 128) to check, but both runs got OOM-killed in my environment before completing, so I can't confirm or rule that out myself. Filing this anyway since the specific numbers below are concrete evidence that might help either confirm #724's cause or reveal something narrower.

### Symptom

The model's output tensor is a softmax layer — mathematically guaranteed to sum to ~1.0 regardless of input. On a real forward pass, it summed to ~23.4 instead. I isolated the graph itself as correct first: the float32 TFLite variant produced in the same run sums to exactly 1.0 on the same input, so the graph/softmax placement is fine.

Both the input and output tensors ended up with `scale=0.00392157 (1/255), zero_point=128` — the same values for both tensors, which looks like a generic default `[-0.5, 0.5]`-ish range assumption rather than real observed statistics from the supplied calibration images. I confirmed the calibration-data reconstruction formula onnx2tf itself uses (`(calib_data - mean) / std`) is correct in isolation for my inputs — applying it directly to my `.npy` file reproduces the intended raw `[0, 255]` range.

### Confirmed via A/B comparison against plain TFLiteConverter

I ran the identical ONNX graph and identical calibration data through plain `tf.lite.TFLiteConverter.from_saved_model(...)` instead (using `flatbuffer_direct_output_saved_model=True` to get a plain SavedModel out of onnx2tf first, then quantizing myself). Same data, same intent, correct result:

- Input tensor calibrated to `scale=1.0, zero_point=0` (exactly right for a raw `[0,255]` byte range)
- Output tensor calibrated to `scale=0.00390625 (1/256), zero_point=0` (correct for a `[0,1]` softmax range)
- Forward pass sums to `1.0000` on both random noise and real images

Also notable: the onnx2tf-produced file was **341MB**, nearly identical to the un-quantized float32 model's 343MB, while the correctly-calibrated file was **87.9MB** — consistent with #724's bloat symptom, not just a miscalibrated tensor.

### How to reproduce

```python
zero = [[[[0.0, 0.0, 0.0]]]]
inv_255 = [[[[1 / 255.0, 1 / 255.0, 1 / 255.0]]]]

onnx2tf.convert(
    input_onnx_file_path="houseplant_vit.onnx",
    output_folder_path="saved_model",
    output_integer_quantized_tflite=True,
    quant_type="per-channel",
    custom_input_op_name_np_data_path=[
        ["pixel_values_0_255", "calibration_data.npy", zero, inv_255],
    ],
    input_quant_dtype="uint8",
    output_quant_dtype="uint8",
)
```
where `calibration_data.npy` is 40 real photos, shape `(40, 224, 224, 3)`, float32, pre-scaled to `[0, 1]`.

### Workaround used

Use onnx2tf only to translate ONNX → a plain TF SavedModel (`flatbuffer_direct_output_saved_model=True, output_integer_quantized_tflite=False`), then do all INT8 quantization directly with plain `tf.lite.TFLiteConverter` and a hand-written `representative_dataset` generator.

**Comment by PINTO0309** [comment]
Originally, there were no plans to support quantization. Accuracy is not guaranteed. I do not intend to actively make improvements going forward.

By the way, this issue is completely unrelated to #724. I didn't properly implement quantization, so a simple Float32 model was generated.

Since `flatbuffer_direct` exists to completely eliminate the dependency on TensorFlow, the policy is to avoid using SavedModel entirely.

https://github.com/PINTO0309/onnx2tf/releases/tag/2.5.1
